react/UnitPrice.tsx

```diff
@@ -44,14 +46,28 @@ const UnitPrice: StorefrontFunctionComponent<UnitPriceProps> = ({
 
   const unitPrice = item[unitPriceType] ?? 0
 
-  return (item.quantity > 1 || unitPriceDisplay === 'always') &&
-    unitPrice > 0 &&
-    !isManualPriceDefined ? (
+  if (
+    !(
+      (item.quantity > 1 || unitPriceDisplay === 'always') &&
+      unitPrice > 0 &&
+      !isManualPriceDefined
+    )
+  ) {
+    return null
+  }
```

Comment:
I think we could benefit from breaking this conditional down to variables, it's kind of cryptic right now :sweatman:

Reply:
ooooooh, but i see what you did. It's better than before, though.